P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -1326,6 +1326,13 @@ in the Eliot typescript mentioned above, there are --> several
                of the <soCalled>overlay</soCalled> levels implicit in the manuscripts of James
                Joyce's <title>Ulysses</title>. A fuller and more complex way of supporting such an
                approach is discussed in <ptr target="#PH-changes"/>. </p>
+            
+            <p>A special case of a substitution may consist of a superfluous word or phrase that is 
```

Comment:
Good. Suggested mildly modified version:
~~~xml
<p>A special case of a substitution may consist of a superfluous word
or phrase that is silently replaced by some addition. E.g. a scribe
abandons a word (without indicating it should be deleted), and then
writes a replacement word immediately after. Here the encoder may
interpret this as an <soCalled>unmarked</soCalled> deletion which can
then be combined with a corresponding addition to a substitution.</p>
~~~